cla_frontend/assets-src/javascripts/app/js/services/models.js

```diff
@@ -316,6 +316,9 @@
       resource.prototype.isInScopeUnknown = function() {
         return (this.state === undefined || this.state === DIAGNOSIS_SCOPE.UNKNOWN);
       };
+      resource.prototype.nonCLACategory = function(){
```

Comment:
All of the other functions here are named in the form "is a thing" or "has a thing". You should follow this naming convention if possible, purely for consistency.

Reply:
Done